### PR TITLE
AB#514, Added command handler when user types commands in GUI

### PR DESCRIPTION
**Pull request by raymond-sde**

### CODE REVIEW

File: src/main/java/com/locallampoon/fiveh/client/FiveHClient.java [hunk 1]
```diff
@@ -9,6 +9,6 @@
 
     public static void main(String[] args) throws IOException {
         game = new Game();
-        game.start();
+        game.startV2();
```
Comment (chaynsaw) [inline src/main/java/com/locallampoon/fiveh/client/FiveHClient.java:12]: Non-blocking, but perhaps you could define or comment this method more descriptively, like so... 
`// V2 initializes the GUI` or `game.startWithGUI`
Just a suggestion.

File: src/main/java/com/locallampoon/fiveh/core/Game.java [hunk 2]
```diff
@@ -7,14 +7,15 @@
 import java.util.Map;
 
 public class Game implements Serializable {
-    private final Player player;
-    private Map<String, Room> houseMap;
+    private static Player player;
```
Comment (chaynsaw) [inline src/main/java/com/locallampoon/fiveh/core/Game.java:10]: For my own personal edification, does making these variables static serve a functional purpose? Like you need to load them before anything else in the class is run/initialized?

**Review by chaynsaw — APPROVED** [review]

**Comment by chaynsaw** [comment]
Just a few general comments, @raymond-sde...
I'm noticing that there's scroll for the narrative text, but only a little bit. From a user perspective, do you think maybe you should do away with the vertical scrollbar and just make the window big enough to accommodate the narrative text? 

Also, I retargeted my stats_panel PR to this branch, after noticing that there would be conflicts from reading this PR. I know you've tried to preserve the original code with its `start` method, but if we're going to a full GUI, perhaps it's better to just remove the code that isn't being used any longer? After all, it's still there in the commit history. The merge conflict wasn't too difficult to deal with, but it would have been more obvious from the jump if you had just removed `start`, and then I'd know I'd have to move anything related to statsPanel into `startV2`. Maybe for the others, this might cause some issues down the line. 

Food for thought. Non-blocking.

File: src/main/java/com/locallampoon/fiveh/core/Game.java (hunk 2 above)
Comment (raymond-sde) [reply src/main/java/com/locallampoon/fiveh/core/Game.java:10]: The intent to make these static was for accessibility outside the Game class without the need to creating a new instance. The ConsolePanel calls Game.handleCommand(), which any dependent variables need to be static in order to be accessed.

**Pull request merged**